plugins/identity/app/javascript/widgets/app_credentials/NewForm.jsx

```diff
@@ -35,6 +36,16 @@ export function NewForm({ onSubmit, onCancel, setError }) {
           }}
         />
       </FormRow>
+      <FormRow>
+        <Checkbox
+          label="Unrestricted"
+          name="unrestricted"
+          helptext="Optional flag to restrict whether the application credential may be used for the creation or destruction of other application credentials or trusts."
```

Comment:
Suggestion for a slightly different wording to make it a bit clearer what happens when set to true:

```suggestion
          helptext="Unrestricted application credentials may be used for the creation or destruction of other application credentials or trusts"
```